Doc/license.rst

```diff
@@ -571,7 +571,7 @@ The :mod:`test_epoll` contains the following notice::
 Select kqueue
 -------------
 
-The :mod:`select` and contains the following notice for the kqueue interface::
+The :mod:`select` contains the following notice for the kqueue interface::
```

Comment:
This still isn't quite right, it now reads `"The select contains the following ..."`, which is nonsensical.  Either remove the leading 'The', or (preferably) add 'module' after `select`.